### ISSUE TITLE
Add option to display colors in the Entities Browser

### ISSUE THREAD

**Issue by hernanmd** [cormas/cormas#26]
Currently the Cormas Entities Browser opened with:

`CormasEntitiesBrowser new buildBrowser`

does not display the color codes from the VisualWorks version to visualize the type of entities (spatial green, social blue, passive orange).

Such coloring option could be added as a Preference, which defaults to use the Green/Blue/Orange or not using it.

**Comment by SergeStinckwich** [comment]
Yes sure the same colours than the VW can be used. I'm not sure we should add preferences for that :-)
